python/ray/tests/test_dashboard_profiler.py

```diff
@@ -0,0 +1,128 @@
+import pytest
+import subprocess
+import os
+import requests
+import sys
+
+import ray
+from ray._private.test_utils import (
+    format_web_url,
+    wait_until_server_available,
+    wait_until_succeeded_without_exception,
+)
+
+
+@pytest.mark.skipif(
+    os.environ.get("RAY_MINIMAL") == "1",
+    reason="This test is not supposed to work for minimal installation.",
+)
+def test_profiler_endpoints(ray_start_with_dashboard):
+    # Sanity check py-spy is installed.
+    subprocess.check_call(["py-spy", "--version"])
```

Comment:
I don't see any code to add py-spy to any requirements.
`py-spy` is already installed in the test machines?

Reply:
Yeah, it's in our requirements.txt, which should be installed on CI (let's see).